controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -201,23 +201,28 @@ def _prepare_mocks_for_object_still_in_use(self):
 
     def test_delete_volume_has_snapshot_fcmaps_not_removed(self):
         self._prepare_mocks_for_object_still_in_use()
+        fcmaps_list_as_target = Mock(as_list=[])
         fcmaps_as_source = self.fcmaps
         fcmaps_as_source[0].copy_rate = "0"
-        self.svc.client.svcinfo.lsfcmap.side_effect = [Mock(as_list=[]), Mock(as_list=fcmaps_as_source)]
+        fcmaps_list_as_source = Mock(as_list=fcmaps_as_source)
+        self.svc.client.svcinfo.lsfcmap.side_effect = [fcmaps_list_as_target, fcmaps_list_as_source]
         with self.assertRaises(array_errors.ObjectIsStillInUseError):
             self.svc.delete_volume("vol")
 
     def test_delete_volume_still_copy_fcmaps_not_removed(self):
         self._prepare_mocks_for_object_still_in_use()
+        fcmaps_list_as_target = Mock(as_list=[])
```

Comment:
I would avoid using the type in the variable name

Reply:
its not about the type, just making sure that we are clear about what its for

Reply:
it can be both clear and without the unnecessary type

Reply:
done